.github/workflows/release.yaml

```diff
@@ -0,0 +1,37 @@
+name: generate-release
+# This workflow will generate changelog and release notes.
+# Source: https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/.github/workflows/release.yml
+
+on:
+  workflow_dispatch:
```

Comment:
Should probably limit this to the `main` branch?

Reply:
that sounds reasonable given how we work today. If we had staging or release lines/branches I could see this working differently.